projects/packages/my-jetpack/_inc/utils/format-percentage.ts

```diff
@@ -0,0 +1,19 @@
+import { numberFormat } from '@automattic/jetpack-components';
+import type { FormatNumberFunction } from './types';
+
+const defaultConfig: Intl.NumberFormatOptions = {
+	style: 'percent',
+};
+
+const formatPercentage: FormatNumberFunction = ( number, config = defaultConfig ) => {
+	// Force percentage
+	config.style = 'percent';
```

Comment:
I imagine we don't need to force the style to be a percentage until after we've checked for invalid number inputs?